CONTRIBUTING.md

```diff
@@ -56,7 +56,7 @@ crystal spec
 
 ## Define plugins
 
-- `github-tags` will work in most cases. We have the scaffold. Run and update diff.
+- `github-tags` or `github-releases` will work in most cases. We have the scaffold. Run and update diff.
```

Comment:
When both will work, preferring to releases will avoid RC, I think... So updating template might be better 🤔